Analysis/Tasks/PWGCF/FemtoDream/femtoDreamProducerTask.cxx

```diff
@@ -149,8 +202,57 @@ struct femtoDreamProducerTask {
                             track.tofNSigmaEl(), track.tofNSigmaPi(), track.tofNSigmaKa(), track.tofNSigmaPr(), track.tofNSigmaDe());
         }
       }
-    }
-  }
+    } //end tracks loop
+
+    for (auto& v0 : fullV0s) {
+      auto postrack = v0.posTrack_as<aod::FilteredFullTracks>();
+      auto negtrack = v0.negTrack_as<aod::FilteredFullTracks>(); ///\tocheck funnily enough if we apply the filter the sign of Pos and Neg track is always negative
+      if (!v0Cuts.isSelectedMinimal(col, v0, postrack, negtrack)) {
+        continue;
+      }
+      v0Cuts.fillQA(col, v0); ///\todo fill QA also for daughters
+      auto cutContainerV0 = v0Cuts.getCutContainer(col, v0, postrack, negtrack);
+      if ((cutContainerV0.at(0) > 0) && (cutContainerV0.at(1) > 0) && (cutContainerV0.at(2) > 0)) {
+        int postrackID = v0.posTrackId();
+        int rowInPrimaryTrackTablePos = -1;
+        for (size_t i = 0; i < tmpIDtrack.size(); i++) {
+          if (tmpIDtrack.at(i) == postrackID) {
+            printf("++++++++++++Equal ID tracks and positive daughter++++++++++++++++++++\n");
+            printf("i= %i --- tmpIDtrack.at(i) = %i --- postrackID = %i\n", i, tmpIDtrack.at(i), postrackID);
+            printf("ptTracks in Femtotable at i= %.2f\n", temptrackPt.at(i));
+            printf("++++++++++++++++++++++++++++++++\n");
+            rowInPrimaryTrackTablePos = i;
+            break;
+          }
+        }
+        childIDs[0] = rowInPrimaryTrackTablePos;
+        childIDs[1] = 0;
+        ROOT::Math::PxPyPzMVector postrackVec(v0.pxpos(), v0.pypos(), v0.pzpos(), 0.);
+        ROOT::Math::PxPyPzMVector negtrackVec(v0.pxneg(), v0.pyneg(), v0.pzneg(), 0.);
+        outputTracks(outputCollision.lastIndex(), postrackVec.Pt(), postrackVec.Eta(), postrackVec.Phi(), aod::femtodreamparticle::ParticleType::kV0Child, cutContainerV0.at(1), 0., childIDs);
+        const int rowOfPosTrack = outputTracks.lastIndex();
+        tempPostrackPt.push_back(postrackVec.Pt());
+        printf("childIDs[0] = %i -- postrackVec.Pt() = %.4f -- temptrackPt = %.4f\n",
+               childIDs[0], static_cast<int>(temptrack[0]), postrackVec.Pt(), temptrackPt.at(childIDs[0]));
+
+        int negtrackID = v0.negTrackId();
+        int rowInPrimaryTrackTableNeg = -1;
+        for (size_t i = 0; i < tmpIDtrack.size(); i++) {
+          if (tmpIDtrack.at(i) == negtrackID) {
+            rowInPrimaryTrackTableNeg = i;
+            break;
+          }
+        }
+        childIDs[0] = rowInPrimaryTrackTablePos;
```

Comment:
This is a bit misleading , can you just replace the entry of the pos track?

Reply:
You mean here the negative track right childIDs[1]?
So in the negative daughter track inputs of the table we only have the corresponding value.

Reply:
yes